ui/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "libretime-ui",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit --testMatch '**/*.spec.js'",
```

Comment:
Figure out if I can get rid of redundantly specifying testMatch here and further down in `"jest"`.